Update o-ran-integration.md (#159)
nephio-project/docs: content/en/docs/network-architecture/o-ran-integration.md

The primary role of the NFO and DMS services is to provide for the lifecycle management of the cloud deployments for O-RAN NFs. These services have identified the following O-Cloud capabilities that are planned to be developed using Nephio capabilities:

- NF Deployment Lifecycle Management
- NF Termination Lifecycle Management

> - 

Reviewer: ```suggestion 
```

{{% alert title="Note" color="primary" %}}

In R3 Nephio use cases and component architectures have been defined for the Deploy O-RAN NF use case.
In R3, NFO supports OAI O-RAN(OCU-CP, OCU-UP, ODU) NF deployments using the REST APIs below.

> ##  input will be packagevariant of OAI NF.

Author: Hi @liamfallon , these are the API's of NFO to trigger OAI NF deployment. In R3 we don't have FOCOM support so NFO talks directly with Porch over the API.

{{% alert title="Note" color="primary" %}}

In R3 Nephio use cases and component architectures have been defined for the Deploy O-RAN NF use case.

> In R3 ,NFO supports OAI O-RAN(OCU-CP,OCU-UP,ODU) NF deployments using below REST API's.

Reviewer: ```suggestion
In R3, NFO supports OAI O-RAN(OCU-CP, OCU-UP, ODU) NF deployments using the REST APIs below.
```

> ##  input will be packagevariant of OAI NF.

Reviewer: ```suggestion
## NFO REST APIs

The input is a PackageVariant of the OAI network functions.
```
